components/docs-chef-io/content/automate/servicenow_reference.md

```diff
@@ -376,3 +377,74 @@ The ServiceNow app deletes these records of the corresponding closed incidents,
 `x_chef_incident.logging.verbosity`
 
 : Used to debug the data in ServiceNow. Enables the selected logging level and is visible in logs. The possible values are `debug`, `warn`, `info`, and `error`. Default: `error`.
+
+## Event Creation App Roles
+
+You can associate a single user with more than one role.
+
+### Role `x_chef_event.admin`
+
+You can assign the `x_chef_event.admin` role to a user other than a System Administrator to allow another user to manage the application properties and logs.
+
+The System Administrator authorization includes access to the tasks in the **Admin** role.
+
+The **Admin** role grants user access to the following:
+
+- Chef Events menu
+- Client Infra Client Run Alerts menu item
+- Chef InSpec scans Alerts menu item
+- Client Infra Client Runs
+- Chef InSpec scans
+- Properties menu item
+- Support menu item
+- Logs menu item
+
+### Role `x_chef_event.chef_inspec_scans_user`
+
+The `x_chef_event.chef_inspec_scans_user` role is suitable for those users who require application access without administration rights. The role grants a user access to the:
+
+- Chef Events menu
+- Chef Infra scans Alerts menu item
+- Chef InSpec scans menu item
+- Support menu item
+
+{{< note >}}
+
+Client run and Chef InSpec scan records are linked to events, and is appropriate to assign this role to users who manage events in ServiceNow.
+
+{{< /note >}}
+
+### Role x_chef_event.api
+
+The `x_chef_event.api` role is suitable for users responsible for integrating the Chef Automate data into ServiceNow. We recommend creating a new user specifically for this role. The Chef Automate Event Creation App requires the API role to set up communication with Chef Automate.
```

Comment:
Simplify and prioritize the sentences.

The `x_chef_event.api` role is:

1. Required to set up the Event Creation App with Chef Automate
2. Necessary for users integrating Chef Automate and ServiceNow
3. The best practice is to create a user for this role